subprojects/kotlin-dsl-integ-tests/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/ProjectSchemaAccessorsIntegrationTest.kt

```diff
@@ -613,6 +613,26 @@ class ProjectSchemaAccessorsIntegrationTest : AbstractPluginIntegrationTest() {
         )
     }
 
+    @Test
+    fun `can access NamedDomainObjectContainer extension content via generated accessor`() {
+        withBuildScript(
+            """
+
+            plugins {
+                java
+                `jvm-test-suite`
+            }
+            
+            testing {
+                suites.test.get().useJUnitJupiter()
```

Comment:
We should test both the generated property and function:
```suggestion
                suites.test.get().useJUnitJupiter()
                suites.test {
                    dependencies {}
                }
```